### PR TITLE
Kicking off project Baru

### PR DESCRIPTION
**Pull request by thomaseizinger**

### CODE REVIEW

File: blog/2021-07-14-project-baru.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+title: "Project Baru: DeFi on Liquid"
+author: thomas
+author_url: https://github.com/thomaseizinger
+author_image_url: https://avatars1.githubusercontent.com/u/5486389
+tags: [bitcoin,defi,liquid,options,futures,loans]
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+<!-- <img alt="Hibernation" src={useBaseUrl('blog/assets/images/2021-07/hibernate_bear_tx_chaining_dream.png')} /> -->
```
Comment (bonomat) [inline blog/2021-07-14-project-baru.md:11]: Image idea 😬

http://harrangueman.blogspot.com/2005/09/schnappi.html

File: blog/2021-07-14-project-baru.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+Fortunately though, Liquid offers tools to build this.
+We already utilised this to build [trustless loans](2021-03-31-borrowing-without-a-middleman.md).
+
+## Project Baru
+
+Project Baru is our effort to continue DeFi on the Liquid platform.
+Our goal is to build all sorts of trustless financial products, including options, futures and potentially even CFDs.
+People will be able to take out loans, buy a future's contract and sell it to someone else, all without having to entrust their funds with a custodian.
+
+In case you are curious about the name: Baru is the name of a [native Australian crocodile species](https://australian.museum/learn/australia-over-time/extinct-animals/baru-darrowi/) that would hunt in fresh water pools (Liquid anyone?) by lurking underneath the water surface, waiting for prey.
```
Comment (bonomat) [inline blog/2021-07-14-project-baru.md:52]: We could be a bit more explicit here and say what we want to build first, i.e. : start with trustless lending first and continue then with option (or future) trading.

**Review by bonomat — APPROVED** [review]
Reads nicely. One thing I'm missing is the call to action to join our awesome matrix chat :)

File: blog/2021-07-14-project-baru.md (hunk 1 above)
Comment (thomaseizinger) [reply blog/2021-07-14-project-baru.md:11]: No thanks :joy:

**Pull request merged**